src/swagger/trips.swagger.js

```diff
@@ -294,3 +295,25 @@
 *       '400':
 *             description: Please double checks your departure and arrival dates.
 * */
+/**
+* @swagger
+* /api/v1/trips/{requestId}/confirm:
+*   put:
+*     tags:
+*       - Trips
+*     name: re-confirm a trip request
+*     summary: manager should be able to re-confirm a trip request decision
+*     produces:
+*       - application/json
+*     consumes:
+*       - application/json
+*     parameters:
+*       - name: token
+*         in: header
+*         description: jwt token of the user
+*       - name: requestId
+*         in: path
+*     responses:
+*       '400':
+*             description: Please double checks your departure and arrival dates.
```

Comment:
We can add a success response too and refine this one if possible.

Reply:
fixed the swagger